### ISSUE TITLE
QuestBook server incompatable

### ISSUE THREAD

**Issue by spannerman79** [Drethic/QuestBook#1]
Forge version doesn't matter, its calling a client side only class on a server

```
cpw.mods.fml.common.LoaderException: java.lang.NoClassDefFoundError: net/minecraft/client/gui/GuiScreen
        at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
        at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:559)
        at cpw.mods.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:88)
        at cpw.mods.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:314)
        at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/gui/GuiScreen
        at drethic.questbook.item.QBItems.<clinit>(QBItems.java:7)
        at drethic.questbook.proxy.CommonProxy.preInit(CommonProxy.java:12)
        at drethic.questbook.proxy.ServerProxy.preInit(ServerProxy.java:11)
        at drethic.questbook.QuestBook.preInit(QuestBook.java:28)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
        at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
        at com.google.common.eventbus.EventBus.post(EventBus.java:275)
        at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
        at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
        at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
        at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
        at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
        at com.google.common.eventbus.EventBus.post(EventBus.java:275)
        at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
        at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
        ... 5 more
```

**Comment by Drethic** [comment]
This issue should now be fixed in 1.0.1.  Please let me know if it is not.

**Comment by spannerman79** [comment]
:+1: works - thanks :smile:
